### PR TITLE
Mega Menu adjusts

### PR DESCRIPTION
**Pull request by paulovmr**
* Apps menu fix;
* Removed deprecated Library monitor.

### CODE REVIEW

**Comment by paulovmr** [comment]
jenkins execute full downstream build

**Comment by paulovmr** [comment]
Part of an ensemble:
* https://github.com/kiegroup/kie-wb-common/pull/1097
* https://github.com/kiegroup/kie-wb-distributions/pull/599

**Comment by paulovmr** [comment]
jenkins retest this

**Comment by paulovmr** [comment]
jenkins retest this

File: kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/menu/DefaultWorkbenchFeaturesMenusHelper.java [hunk 1]
```diff
@@ -208,12 +208,8 @@ public void addUtilitiesMenuItems() {
 
         TopLevelMenusBuilder<MenuBuilder> menuBuilder;
 
-        if (hasAccessToPerspective(APPS)) {
-            menuBuilder = MenuFactory.newTopLevelCustomMenu(iocManager.lookupBean(AppsCustomMenuBuilder.class).getInstance()).endMenu();
-            menuBuilder.newTopLevelCustomMenu(iocManager.lookupBean(CustomSplashHelp.class).getInstance()).endMenu();
-        } else {
-            menuBuilder = MenuFactory.newTopLevelCustomMenu(iocManager.lookupBean(CustomSplashHelp.class).getInstance()).endMenu();
-        }
+        menuBuilder = MenuFactory.newTopLevelCustomMenu(iocManager.lookupBean(AppLauncherMenuBuilder.class).getInstance()).endMenu();
```
Comment (ederign) [inline kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/menu/DefaultWorkbenchFeaturesMenusHelper.java]: no check for permissions?

Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/menu/DefaultWorkbenchFeaturesMenusHelper.java]: The App launcher does not need permissions to be added. It displays (when not empty) a dropdown with added launchers (they are added [here](https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/popups/launcher/AppLauncherPresenter.java#L46)). This way, the code responsible to add launchers here should check any necessary permissions.

**Pull request merged**